components/ntp_background_images/browser/ntp_sponsored_images_data.h

```diff
@@ -55,8 +55,12 @@ struct Logo {
   ~Logo();
 };
 
+enum WallpaperType { kImage, kRichMedia };
+
 struct SponsoredBackground {
-  base::FilePath image_file;
+  WallpaperType wallpaper_type;
+  base::FilePath file_path;
+  GURL wallpaper_url;
```

Comment:
As discussed lets remove and build when needed.

Reply:
discussed in DM and decided to renamed to `url`